cmd/up/up_test.go

```diff
@@ -102,3 +102,86 @@ func Test_printDisplayContext(t *testing.T) {
 	}
 
 }
+
+func TestEnvVarIsAddedProperlyToDevContainerWhenIsSetFromCmd(t *testing.T) {
+	var tests = []struct {
+		name                    string
+		dev                     *model.Dev
+		upOptions               *UpOptions
+		expectedNumManifestEnvs int
+	}{
+		{
+			name:                    "Add only env vars from cmd to dev container",
```

Comment:
Can we add a test with non ascii characters? Would you mind adding a test with some strange characters (f.e. `~$#@` etc)